CHANGELOG.md

```diff
@@ -35,6 +35,8 @@ FEATURES:
 * `r/tfe_team_project_access`: Add `variable_sets` attribute to `project_access`, by @mkam [#1565](https://github.com/hashicorp/terraform-provider-tfe/pull/1565)
 
 BUG FIXES:
+* `r/tfe_oauth_client`: Ensure `oauth_token_id` updates register when performing a `terraform apply`, by @hashimoon[#1631](https://github.com/hashicorp/terraform-provider-tfe/pull/1631)
```

Comment:
this seems to be under the previously released version `## v.0.64.0`, it should move to the `Unreleased` section